src/components/polygons/boundary-drawer.vue

```diff
@@ -251,6 +297,14 @@ $crosshair2: url('/assets/cursors/crosshair2.svg'), crosshair;
 
 .draw-boundary {
     cursor: $crosshair;
+
+    &.disabled {
+        cursor: wait;
+        rect {
+            fill: #f8f9fa !important;
```

Comment:
do we really need !important. I always worry about that, but probably because I am not so confidant in CSS.